backend/django/core/utils/util.py

```diff
@@ -98,7 +98,7 @@ def upload_data(form_data, project, queue=None, irr_queue=None, batch_size=30):
             batch_size=batch_size,
```

Comment:
I'm noticing the default batch size is back to "number of labels x 10" in the project creation form. Not sure if that change was in a defunct branch, or this branch needs npsas_main merged in?

Reply:
Good catch! I'm not sure what happened to that change, but I added it again here on this branch